uktrade/great-cms
# Feature/kls 684 export academy signin

New Sign in page for Aventri users with a Great account migrating to Great Export Academy from Aventri

**Context**
This PR adds the first version of the export academy signin flow. In this version, the user is already registered for the export academy via Aventri and also has a Great account, and has received a unique link where their email address will be pre-populated and they just need to sign in

The backend implementation of the unique link has not yet been created, so the view currently relies on a registration uuid in the query parameters of the URL. Once the backend has been implemented this will be updated accordingly.

**Reusing IOO code**
This implementation reuses the logic used by international online offer's login view which uses django, rather than the main login flow using react. Currently

Some IOOLogin code has been moved to sso_mixins

**To test**
- Make sure you have an EA registration for your chosen email in your local db
- Make sure you have directory-forms-api, directory-sso and directory-sso-proxy running locally
- Go to `export-academy/signin?registration-id=<your_registration_id>`
- Your email address should be pre-populated
- You should be able to go through the signin

Screenshot

<img width="1405" alt="Screenshot 2023-06-22 at 15 29 23" src="https://github.com/uktrade/great-cms/assets/56790506/62e6ea54-fa23-4f81-a574-a394ccb61cfc">

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-684


### Housekeeping

### Merging
